### ISSUE TITLE
Epic: API database AWS commercial migration and postgres 13 upgrade

### ISSUE THREAD

**Issue by patphongs** [fecgov/fec-epics#209]
**What we're after:**
In order to optimize and maintain database speed and performance, we need to migrate the API database to AWS commercial. At the same time, we need to upgrade our API database postgres from version 10 to 13.

Reference: [Checklist to move to Aurora](https://github.com/fecgov/openFEC/issues/4297#issuecomment-620378893)

## Tickets
- [x] fecgov/openFEC#4993

**Comment by dzhang-fec** [comment]
Thanks @patphongs. I just added this as a tech note on the pg version 10.14 to 13.4 (lastest Aurora pg13) upgrade.

checklist Reference:
1)Recently we have upgraded the psycopg2-binary 2.9.1 from psycopg2-binary 2.7.4 (will do by Dev team)
https://github.com/fecgov/openFEC/blob/develop/requirements.txt
  which looks fine but we need to run loading tests against the new  Aurora pg 13.4.
  (the lastest one is psycopg2-binary 2.9.2 BTW)

2)Aurora PG Database extentions changes requirements at database side (will do by DBA team).
---new 13.4---- extension versions 
"plpgsql"	"1.0"
"aws_commons"	"1.2"
"aws_s3"	"1.1"
"btree_gin"	"1.3"
"dblink"	"1.2"
"pg_prewarm"	"1.2"
"pgcrypto"	"1.3"
"pglogical"	"2.4.0"
"postgres_fdw"	"1.0"
"sslinfo"	"1.2"
"unaccent"	"1.1"
"uuid-ossp"	"1.1"
"plv8"	"2.3.15"
"pg_stat_statements"	"1.8"
"pg_trgm"	"1.5"

---current before upgrade 10.14
SELECT extname, extversion  FROM pg_extension;
"plpgsql"	"1.0"
"aws_commons"	"1.1"
"aws_s3"	"1.1"
"plv8"	"2.3.14"
"btree_gin"	"1.2"
"dblink"	"1.2"
"pg_prewarm"	"1.1"
"pg_stat_statements"	"1.6"
"pg_trgm"	"1.3"
"pgcrypto"	"1.3"
"pglogical"	"2.2.2"
"postgres_fdw"	"1.0"
"sslinfo"	"1.2"
"unaccent"	"1.1"
"uuid-ossp"	"1.1"
